<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductView</class>
 <widget class="QDialog" name="AddProductView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="productName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="preferredVendor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Preferred Vendor</string>
   </property>
  </widget>
  <widget class="QComboBox" name="activeLevel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Activity Level</string>
   </property>
   <item>
    <property name="text">
     <string>Inactive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Out of Stock</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Discontinued</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="productLastCost">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Product Last Cost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productPrice">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Product Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productUPC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Product UPC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addProduct">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
